tests/Makefile.am

```diff
@@ -115,6 +116,30 @@ test_update_refspec_checkpoint_LDFLAGS = $(test_ldflags)
 test_update_refspec_checkpoint_LDADD = $(test_ldadd)
 test_update_refspec_checkpoint_SOURCES = test-update-refspec-checkpoint.c
 
+eos-updater-generated.c: eos-updater-generated.h
+	cp ../src/eos-updater-generated.c .
+
+eos-updater-generated.h:
+	cp ../src/eos-updater-generated.h .
```

Comment:
@dbnicholson In the end just adding the `eos-updater-generated.*` files to the `nodist_test_update_direct_SOURCES` didn't solve the problem for distclean. So I added these changes instead. I am sure this is not the most elegant way to do it, so if you know the right way, please let me know.

Reply:
I’ve just pushed a fixup which rearranges this to link against `libeos-updater-dbus.la` instead, which builds for me, though you should probably test it.